### PR TITLE
added taylor series

### PR DESCRIPTION
**Pull request by mehradans92**

### CODE REVIEW

File: data/taylor_series/taylor_series.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        coef = (-1)**i
+        num = x**(2*i)
+        denom = math.factorial(2*i)
+        cos_approx += (coef) * ((num)/(denom))
+    return cos_approx
+
+x = math.radians(45)
+n = 8
+
+result = True if np.isclose(taylor_series(func_cos, x, n), np.cos(x)) else False
```
Comment (navneeth3005) [inline data/taylor_series/taylor_series.py:16]: Should'nt the result be `np.isclose(taylor_series(np.cos, x, n), func_cos(x))` ? I am assuming you want to compare the output from your function (which computes the taylor series approximation) against the actual taylor series approximation you have computed for the cos function.

**Review by navneeth3005 — COMMENTED** [review]
Address the comments

File: data/taylor_series/taylor_series.py (hunk 1 above)
Comment (ziyueyang37) [reply data/taylor_series/taylor_series.py]: I agree with Nav, maybe should be `np.isclose(taylor_series(np.cos, x, n), func_cos(x, n))`

**Review by ziyueyang37 — APPROVED** [review]

**Review by navneeth3005 — COMMENTED** [review]
Looks good.

**Pull request merged**